{{cookiecutter.project_name}}/next_steps.md

```diff
@@ -18,7 +18,7 @@ git remote add origin {{ cookiecutter.repository }}
 ## Push the initial commit to a new repo on GitHub
 
 Go to
-[https://github.com/{{cookiecutter.github_organization}}/repositories/new](https://github.com/{{cookiecutter.github_organization}}/repositories/new)
+[https://github.com/{{cookiecutter.github_organization}}?tab=repositories](https://github.com/{{cookiecutter.github_organization}}?tab=repositories)
```

Comment:
Had to reintroduce this format of the link because /repositories/new doesnt seem to work when owner is a user as opposed to an organization

Reply:
For me https://github.com/organizations/sverhoeven/repositories/new works, does https://github.com/organizations/jspaaks/repositories/new not work for you?

Reply:
O I see I didn't have the /organizations/ in between, reverting once more